portal/src/locale-data/en.json

```diff
@@ -276,6 +276,8 @@
   "PasswordlessAuthenticatorScreen.subsequent-logins.sms-body.label": "SMS body for subsequent logins",
 
   "OAuthClientConfiguration.title": "Client Applications",
+  "OAuthClientConfigurationScreen.api-endpoint.header": "API Endpoint",
+  "OAuthClientConfigurationScreen.api-endpoint.desc": "Configure Client ID to be used at the API endpoint: {a, react, href{apiEndpoint} children{{apiEndpoint}}}. You can change this endpoint at the DNS section of the page with your custom domains.",
```

Comment:
I think apiEndpoint should be text instead of link? As it links to setting page instead of page in portal app (redirect from `/` to `/setting`). If this is intended, should add `_blank`.

Reply:
The endpoint should be a link.

`You can change this endpoint at the [DNS settings] with your custom domains.` Make `[DNS settings]` a link to the DNS page.

Reply:
The layout here is a bit confusing. I suggest we layout as follows:

```
# Client Applications

## Allowed origins
Configure cross-origin...

## Endpoint and client applications
Configure client applications to be used at the endpoint:
[Endpoint link] [line break]
You can change this endpoint at the [DNS settings] with your custom domains.

[The client list here]
```